cmd/rqlite/main.go

```diff
@@ -721,6 +727,67 @@ func createHostList(argv *argT) []string {
 	return hosts
 }
 
+// parseRqliteHostEnv parses the RQLITE_HOST environment variable and updates
+// the connection parameters in the argv struct if command-line flags weren't explicitly set.
+// The format is [scheme://]host[:port]
+// Returns true if the environment variable was parsed successfully, false otherwise.
+func parseRqliteHostEnv(argv *argT) bool {
+	envHost := os.Getenv("RQLITE_HOST")
+	if envHost == "" {
+		return false
+	}
+
+	var scheme, host, port string
+
+	// First, try to use url.Parse from the standard library
+	urlStr := envHost
```

Comment:
No, don't do this. Just parse envHost. Instead check later if parsedURL.Scheme is "". If it is, then set scheme to "http".